### PR TITLE
Changed invalid tag > "free-tier"

### PR DESCRIPTION
**Pull request by AnnikaGonnermann**

### CODE REVIEW

File: tutorials/btp-free-tier-account/btp-free-tier-account.md [hunk 1]
```diff
@@ -13,25 +13,23 @@ primary_tag: software-product>sap-business-technology-platform
...
 
 ## How Does Free Tier Work?
 The global account you will create in this tutorial gives you access to a productive environment with both free and paid plans. If you activate **free tier** plans only, you can try out our platform for free, without time limits. You can then easily upgrade to a paid plan without losing your data or work. Creating an instance using a paid plan will incur costs, however. For more information, see [Enterprise Accounts](https://help.sap.com/products/BTP/65de2977205c403bbc107264b8eccf4b/171511cc425c4e079d0684936486eee6.html?locale=en-US&version=Cloud) and [Using Free Service Plans](https://help.sap.com/products/BTP/65de2977205c403bbc107264b8eccf4b/524e1081d8dc4b0f9d055a6bec383ec3.html?locale=en-US&version=Cloud).
```
Comment (jmmargo) [inline tutorials/btp-free-tier-account/btp-free-tier-account.md:19]: I recommend that you place this in the Intro section.  The word "Intro" does not appear on the web page, it is just the marker for the section before the first step.

**Review by jmmargo — CHANGES_REQUESTED** [review]

File: tutorials/btp-free-tier-account/btp-free-tier-account.md (hunk 1 above)
Comment (AnnikaGonnermann) [reply tutorials/btp-free-tier-account/btp-free-tier-account.md:19]: @jmmargo  I'd rather keep the information on credit card together if that's possible.

Comment (MichaelCzcz) [reply tutorials/btp-free-tier-account/btp-free-tier-account.md:19]: It make sense to me to keep it there. But please change the header to sentence case.
There are also some conflicts.

Comment (AnnikaGonnermann) [reply tutorials/btp-free-tier-account/btp-free-tier-account.md:19]: The conflicts come from me changed the tags, right? Okay, I thought I had changed the header yesterday but apparently not. Okay, how does it make sense to approach this logically?

**Comment by MichaelCzcz** [comment]
Please resolve the conflicts.

**Pull request merged**